### DISCUSSION TITLE
Mission Manager python system not starting

### DISCUSSION THREAD

**Discussion by connorjak** [SimDynamX/SpaceTeamsPRO#45] [2024 Lunar SAR Competition Q&A] [answered]
(removed previous issue duplicate details)

**Comment by erSuyognp** [comment]
Thankyou for helping. I have one more issue:
I tried running the starter sim we are provided(without editing anything). All I get to see is these error message and no movement. Isn't it supposed to follow a movement because there are some command given in exampleMM.py
![image](https://github.com/user-attachments/assets/b1378d2e-ec59-4ced-bc3e-7482e7ca35b9)

**Comment by connorjak** [comment]
You can ignore the material and RotateX errors/warnings; those are left over from some last-minute changes to how model import works.

The sim is supposed to start with waiting for 5 seconds then the rover moving to three different waypoints. The rover may have already finished moving; you can scroll down in the log there and see the OnScreenLogMessage messages from the Mission Manager / LTV Entity Behavior

**Comment by erSuyognp** [comment]
I thought the same, I tried to run sim in -2x but nothing was running. There is nothing except the error in the OnScreenLogMessage. Here is the log file if it helps
[Sim_STU2_SAR_Task1.log](https://github.com/user-attachments/files/17536153/Sim_STU2_SAR_Task1.log)
[Sim_STU2_SAR_Task1_PySys_CompetitionBackend.log](https://github.com/user-attachments/files/17536154/Sim_STU2_SAR_Task1_PySys_CompetitionBackend.log)
[Sim_STU2_SAR_Task1_PySys_CompetitionBackend-BACKUP-.log](https://github.com/user-attachments/files/17536155/Sim_STU2_SAR_Task1_PySys_CompetitionBackend-BACKUP-.log)
[Sim_STU2_SAR_Task1_PySys_EntityBehavior_LTV1.log](https://github.com/user-attachments/files/17536156/Sim_STU2_SAR_Task1_PySys_EntityBehavior_LTV1.log)
[Sim_STU2_SAR_Task1_PySys_EntityBehavior_LTV1-BACKUP-.log](https://github.com/user-attachments/files/17536157/Sim_STU2_SAR_Task1_PySys_EntityBehavior_LTV1-BACKUP-.log)
[Sim_STU2_SAR_Task1_PySys_EntityBehavior_LTV2.log](https://github.com/user-attachments/files/17536158/Sim_STU2_SAR_Task1_PySys_EntityBehavior_LTV2.log)
[Sim_STU2_SAR_Task1_PySys_EntityBehavior_LTV2-BACKUP-.log](https://github.com/user-attachments/files/17536159/Sim_STU2_SAR_Task1_PySys_EntityBehavior_LTV2-BACKUP-.log)
[Sim_STU2_SAR_Task1-BACKUP-.log](https://github.com/user-attachments/files/17536160/Sim_STU2_SAR_Task1-BACKUP-.log)

**Comment by connorjak** [comment]
Missing the log for the MissionManager python system. Did it not run?

**Comment by connorjak** [comment, marked as answer]
Your sim config still is pointing to the STU_Competition2 folder for the files, but it looks like instead of changing the paths in the sim config to your repo's default folder name, you changed the folder name to match `STU_Competition2`? It might work for now but this could disrupt things later when submitting. 

I'm not sure exactly what the issue was besides something is going wrong when starting the mission manager python system and it doesn't connect to the sim. If you can run this sim through Compute_Server by command line (see How to Start a Sim wiki page), the console / python output from starting / crashing on that python system may show up there, mixed in with the Logger messages.

**Reply by erSuyognp** [reply]
It worked!!
I deleted the repo and installed it again and this time named it with my Team Name(it worked). 
Thanks,
Team Moonshot Mavericks

**Reply by connorjak** [reply]
Great! I like your team name BTW

**Comment by erSuyognp** [comment]
Yeah it didn't.
